src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,301 @@
+            new ActionArgument("player", ActionArgumentType.PLAYER, ActionArgumentType.Is.OPTIONAL)
+            )),
+    BACKUP(new ActionMenu(
+            "Backup",
+            "backup",
+            ForgePermissionNodes.BACKUP,
+            BackupCommand.class
+            )),
+    HOME_TELEPORT(new ActionMenu(
+            "HomeTeleport",
+            "home",
+            ForgePermissionNodes.HOME_TELEPORT,
+            HomeTeleportCommand.class
+            )),
+    RAID_ALL_TONIGHT(new ActionMenu(
+            "RaidAllTonight",
+            "raid-tonight",
+            ForgePermissionNodes.RAID_ALL_TONIGHT,
+            RaidAllTonightCommand.class
+            )),
+    RAID_ALL_NOW(new ActionMenu(
+            "RaidAllNow",
+            "raid-now",
+            ForgePermissionNodes.RAID_ALL_NOW,
+            RaidAllNowCommand.class
+            )),
+    CHECK_FOR_AUTO_DELETES(new ActionMenu(
+            "CheckForAutoDeletes",
+            "check",
+            ForgePermissionNodes.CHECK_FOR_AUTO_DELETES,
+            CheckForAutoDeletesCommand.class,
+            new ActionArgument("confirmDelete", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.OPTIONAL)
+            )),
+    WHO_AM_I(new ActionMenu(
+            "WhoAmI",
+            "whoami",
+            ForgePermissionNodes.WHO_AM_I,
+            WhoAmICommand.class
+            )),
+    WHERE_AM_I(new ActionMenu(
+            "WhereAmI",
+            "whereami",
+            ForgePermissionNodes.WHERE_AM_I,
+            WhereAmICommand.class
+            )),
+
+    LIST_COLONIES(new ActionMenu(
+            "ListColonies",
+            "list",
+            ForgePermissionNodes.LIST_COLONIES,
+            ListColoniesCommand.class,
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.OPTIONAL),
+            new ActionArgument("abandonedSinceTimeInHours", ActionArgumentType.INTEGER, ActionArgumentType.Is.OPTIONAL)
+            )),
+    REQUEST_SYSTEM_RESET_ALL(new ActionMenu(
+            "RequestSystemResetAll",
+            "rsResetAll",
+            ForgePermissionNodes.REQUEST_SYSTEM_RESET_ALL,
+            RSResetAllCommand.class
+            )),
+
+    LIST_CITIZENS(new ActionMenu(
+            "ListCitizens",
+            "list",
+            ForgePermissionNodes.LIST_CITIZENS,
+            ListCitizensCommand.class,
+            new ActionArgument("colony", ActionArgumentType.COLONY, ActionArgumentType.Is.REQUIRED),
+            new ActionArgument("page", ActionArgumentType.INTEGER, ActionArgumentType.Is.OPTIONAL)
+        )),
+    KILL_CITIZEN(new ActionMenu(
+            "KillCitizen",
+            "kill",
+            ForgePermissionNodes.KILL_CITIZEN,
+            KillCitizenCommand.class,
+            new ActionArgument("colony", ActionArgumentType.COLONY, ActionArgumentType.Is.REQUIRED,
+                    new ActionArgument("citizen", ActionArgumentType.CITIZEN, ActionArgumentType.Is.REQUIRED))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "citizen" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1192)